include/triton/Dialect/TritonGPU/IR/TritonGPUAttrDefs.td

```diff
@@ -516,6 +516,7 @@ For example, the matrix L corresponding to blockTileSize=[32,16] is:
     "unsigned":$versionMinor,
     ArrayRefParameter<"unsigned">:$warpsPerCTA,
     "CTALayoutAttr":$CTALayout,
+    StringRefParameter<"">:$inputType,
```

Comment:
Can you please list all the valid values of this `inputType` parameter? Thanks

Reply:
Why do we need this in the layout?

Reply:
let me try to remove inputType here.

Reply:
Maybe I need change getSizePerThread to getSizePerThread(Type),
and getSizePerCTA, getSizePerCTATile in the same way.

Reply:
So the `inputType` is like `f16` / `f32`?

Reply:
if I can remove inputType, then mmas with different input will not trigger convert.

Reply:
I cannot remove inputType from mma encodeing, because of here:
https://github.com/openai/triton/blob/main/include/triton/Dialect/TritonGPU/IR/TritonGPUAttrDefs.td#L38
and here:
https://github.com/openai/triton/blob/main/lib/Conversion/TritonGPUToLLVM/TypeConverter.cpp#L164

Reply:
would you mind explaining in more detail?